packages/bpk-component-table/src/BpkTableHead.js

```diff
@@ -19,7 +19,7 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
-const BpkTableHead = props => <thead>{props.children}</thead>;
+const BpkTableHead = props => <thead {...props} />;
```

Comment:
https://github.com/facebook/react/blob/master/src/isomorphic/classic/element/ReactElement.js#L214-L230 it gets added to props when passed in third argument